postcss.config.js

```diff
@@ -1,5 +1,7 @@
 module.exports = {
   plugins: [
+    require('tailwindcss'),
+    require('autoprefixer'),
```

Comment:
I think there is some intention on add `autoprefixer` at last like https://tailwindcss.com/docs/using-with-preprocessors because the plugins will run sequentially _(I maight be wrong tho, but if you have the same doubt than me I would move this `autoprefixer` to last)_

Reply:
Actually seems we can remove `autoprefixer` all together given this link on the tailwindcss docs https://tailwindcss.com/docs/using-with-preprocessors#future-css-features. I'll remove it and just leave `postcss-preset-env`